### PR TITLE
wiki: update client install and config page

### PR DESCRIPTION
**Pull request by chappjc**
This contains a number of updates to the Client Installation and Configuration page to make it clearer, more concise, and up-to-date with the latest release.

One notable change is the wallet support summary:

image

The red "X" has since been changed to a more subtle black "x".  See the DEX matrix chat for motivation.

### CODE REVIEW

File: docs/wiki/Client-Installation-and-Configuration.md [hunk 1]
```diff
@@ -138,50 +122,57 @@ wallet.
...
-
-8. Once the wallet is synchronized and has at least enough to pay the server's
-   defined fee, the form will update, and you should click the button to submit
-   the registration request and transmit the fee amount.
+   like to your new wallet since *only* the amount required for the bond will be
+   spent in the next step. The remaining amount will be in your available
+   balance. For example, you can send yourself 5 BTC and only the required
+   amount will be spent to create the bond in the next step, with the remainder
+   in the wallet's balance, which can then be traded, sent, or simply held in
+   the wallet.
```
Comment (JoeGruffins) [inline docs/wiki/Client-Installation-and-Configuration.md:144]: Some will be locked for future bonds?

**Review by JoeGruffins — APPROVED** [review]

**Review by buck54321 — APPROVED** [review]
Obviously will need some changes after #2385 is in, but this is g2g with @JoeGruffins approval.

**Comment by chappjc** [comment]
> Obviously will need some changes after #2385 is in

Good point.  But for practical reasons, the wiki should generally represent what's in the latest release that most users will see.  So as 1.0 nears we should update imo

**Comment by chappjc** [comment]
wiki subtree deployed https://github.com/decred/dcrdex/wiki/Client-Installation-and-Configuration

**Pull request merged**